frontend/script/gh_deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+git config --global user.name "Travis CI"
+git config --global user.email "noreply+travis@fossasia.org"
+
+export DEPLOY_BRANCH=${DEPLOY_BRANCH:-development}
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/badgeyay" -o "$TRAVIS_BRANCH" != "$DEPLOY_BRANCH" ]; then
+    echo "Skipping deploy; The request or commit is not on development:)"
+    exit 0
+fi
+
+ember deploy gh-pages
```

Comment:
Sorry for not mentioning earlier. Okay, won't we need a commit?

Reply:
@abishekvashok  I have made depoy.js which will handle all these things.

Reply:
I guess it is checking for the branch, is the branch is not `dev` then there is no need to deploy again?
Ping @abishekvashok